docs/source/table.rst

```diff
@@ -1,251 +1,303 @@
 .. list-table::
-   :widths: 5 5 5 5 5 5
+   :widths: 4 4 3 1 1 3 5 3
```

Comment:
I think it will be better to switch the Soundscape column width to <=4. The thing is that on a 13" display the table overflows just a bit. Maybe you have already tried that, in that case let me know.

Reply:
Also, I think it will be great if we can fix the vertical-alignment (https://sublime-and-sphinx-guide.readthedocs.io/en/latest/tables.html). I think it's proably a problem of `*` and `-` placement?